includes/object-cache.php

```diff
@@ -385,7 +385,11 @@ public function __construct( $fail_gracefully = true ) {
                 case 'phpredis':
                     $this->connectUsingPhpRedis( $parameters );
                     break;
+                case 'credis':
+                    $this->connectUsingCredis( $parameters );
+                    break;
```

Comment:
Should we default to Credis in v2, or leave the switch for v2.1.0?

Reply:
Well if we get all connection parameters tested we could switch but it would be better to give the user the option to test `credits` first (and potentially report problems).